src/sentry/notifications/activity/base.py

```diff
@@ -17,7 +18,7 @@
 from sentry.types.integrations import ExternalProviders
 
 
-class ActivityNotification(BaseNotification):
+class ActivityNotification(BaseNotification, ABC):
```

Comment:
what is ABC and why is it needed here?

Reply:
[ABC](https://docs.python.org/3/library/abc.html) is python's way of marking a class as abstract. That means that 
```
notification = ActivityNotification(...)
```
is illegal, but you can still instantiate objects for classes that inherit from it.